Assets/Scripts/CoinTacker/CoinCounter.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Written by Rebecca Henry
+// 4/23/21
+using UnityEngine;
+using UnityEngine.UI;
+
+public class CoinCounter : MonoBehaviour
+{
+    public Text CoinCount;
+    // Players current coin count
+
+    void Update()
+    {
+        CoinCount.text = "0"; // player preferences for score
```

Comment:
Are you going to update this to check the player's coin count thus far from selling? Or will Ellie be doing that?